documentation/API_documentation/API_definition.md

```diff
@@ -0,0 +1,192 @@
+# Reference
+CAMARA Number Verify is based on Mobile Connect Verified MSISDN standardized product:
+
+[IDY.54](assets/docs/(placeholder4MCspec).docx)
+
+**IDY.54 Mobile Connect Verified MSISDN Definition and Technical Requirements** 
+
+## About Mobile Connect
+
+Mobile Connect is a worldwide initiative by Mobile Operators to bring a wide portfolio of Identity services to market that enable SPs and End-Users to transact with one-another more securely through strong authentication, authorisation and exchange of attributes, subject to User consent.
+
+The Mobile Connect architecture consists of a Core framework around which additional components can be added to support the different Mobile Connect services as well as other API based services that delivers value to the customer. The Core framework is based upon the OpenID Connect (OIDC) protocol suite and allows Users to be identified by their MSISDN (or a related Pseudonymous Customer Reference) to enable authentication on their mobile device. Attributes sharing services are based upon the OAUTH2.0 framework - Client Credentials grant.
+
+## Number Verify / Verified MSISDN definition
+
+Mobile Connect Verified MSISDN allows Developer // Service Provider (SP) to verify the MSISDN (phone number) of the device connected to the mobile data network through which a User is accessing an SP service. In doing so, it enables the SP to check that the device being used to access a particular SP User account belongs to the account holder. Such a service can also be used by theSP for verifying the User’s device as a complementary service when a User is being authenticated.
+
+Mobile Connect Verified MSISDN is defined as two service variants:
+
+- **Verified MSISDN Match**: in which the Mobile Operator compares the MSISDN associated with the mobile device against that provided by the SP in the service request1. The MSISDN can be supplied in an E164 format [5] or in a hashed form. Verified MSISDN Match ensures no data is shared by the Mobile Operator.
+- **Verified MSISDN Share**: in which the Mobile Operator provides the mobile device MSISDN to the SP who can then perform the check itself
```

Comment:
This share function is not provided in the swagger - It should be clearly indicated I guess or added in the swagger.